README.md

```diff
@@ -1 +1,50 @@
-# rawhide-rpmdeplint-pipeline
+# rpmdeplint-pipeline
+
+This repository contains definition of the rpmdeplint CI pipeline.
+
+This pipeline runs [rpmdeplint](https://github.com/fedora-ci/rpmdeplint-image) tool on RPM packages and publishes results via [CI Messages](https://pagure.io/fedora-ci/messages).
+
+> ** **Important:** ** Structure of the branches in this repository resembles braches in dist-git, i.e. master branch contains the pipeline definition for Fedora Rawhide, [f32](https://github.com/fedora-ci/rpmdeplint-pipeline/tree/f32) branch is for Fedora 32, etc.
+
+## Test definition
+
+The generic rpmdeplint test is described in a [Flexible Metadata Format](https://pagure.io/fedora-ci/metadata).
+
+The actual definition lives in the [rpmdeplint.fmf](./rpmdeplint.fmf) file.
+
+## Test execution
+
+The pipeline delegates the test execution to the Testing Farm (TODO: link once available). It only collects, archive and report results.
+
+However, it is possible to run the rpmdeplint test locally. To do that, clone this repository and run following command:
+
+```shell
+tmt run -a execute --script "/rpmdeplint/run_rpmdeplint.py -r f32 -t 43617203"
+```
+
+The command above will run rpmdeplint on a Koji build with the [task Id "43617203"](https://koji.fedoraproject.org/koji/taskinfo?taskID=43617203) (f32-backgrounds-32.1.3-1.fc32) and it tests it in the context of Fedora 32.
+
+## Promoting new rpmdeplint image to production
+
+The [rpmdeplint.fmf](./rpmdeplint.fmf) file references a specific version of the container image which contains the rpmdeplint utility and in which the tests actually run in CI.
```

Comment:
Re-worded a bit to clarify the ending as I understand it, though we can discuss a bit more if I misunderstood =) 

"The [rpmdeplint.fmf](./rpmdeplint.fmf) file references a specific version of the container image which contains the rpmdeplint utility, as well as which of the tests to actually run in CI."

Reply:
Hmm, yeah, you're right. I think we could actually omit that second part entirely:

The [rpmdeplint.fmf](./rpmdeplint.fmf) file references a specific version of the container image which contains the rpmdeplint utility.

WDYT?

Reply:
Maybe add "contains the rpmdeplint utility and the required dependencies to run it" if we're pulling in things besides only that package. But if we basically use only base image Linux + the util, it's probably too wordy. :) I'm good either way.

Reply:
We only use the base image + rpmdeplint, no extra deps there :)